src/panels/menu/help/HelpCheatSheet.tsx

```diff
@@ -1,23 +1,24 @@
 import React from 'react';
 import {ProjectSettings} from "../../../config/Variables";
-import {helpModal} from "../../../locale/help";
+import {LocaleHelp} from "../../../config/Locale";
 
 export default class HelpCheatSheet extends React.Component {
 
 	render() {
+		let key = 0;
 		return (<div>
-			<h4>{helpModal[ProjectSettings.viewLanguage].cheatSheet.title}</h4>
-			<p>{helpModal[ProjectSettings.viewLanguage].cheatSheet.t1}</p>
-			{Object.keys(helpModal[ProjectSettings.viewLanguage].cheatSheet.interactions).map((interaction: string) => {
-				return <div><h5>{interaction}</h5>
-					<ul>
-						{Object.keys(helpModal[ProjectSettings.viewLanguage].cheatSheet.interactions[interaction])
-							.map(part => <li><strong>{part + " "}</strong>
-								{helpModal[ProjectSettings.viewLanguage].cheatSheet.interactions[interaction][part]}
+			<h4>{LocaleHelp[ProjectSettings.viewLanguage].cheatSheet.title}</h4>
+			<p>{LocaleHelp[ProjectSettings.viewLanguage].cheatSheet.t1}</p>
+			{Object.keys(LocaleHelp[ProjectSettings.viewLanguage].cheatSheet.interactions).map((interaction: string) =>
+				<div key={key++}><h5 key={key++}>{interaction}</h5>
```

Comment:
It would be better to use `index` for the key, which you can get as a second argument of the mapping function, i.e.:
```
Array.map((interaction: string, index: number) => <div key={index}> ...
```
Because when it is like this, you mutate a variable which is outside of the mapping function, which works but in general it's best to avoid similar side effects / mutate stuff.

Reply:
Implemented.